### PR TITLE
Monkey cubes are now baymed tiers of lethal if you eat them raw as a carbon based lifeform.

### PR DESCRIPTION
**Pull request by silicons**
:^)

### CODE REVIEW

**Comment by yourdoom9898** [comment]
60 brute for being chestburster'd seems low

**Comment by IrkallaEpsilon** [comment]
I wonder how this can be weaponized efficiently...

**Comment by Poojawa** [comment]
vore nerf.

**Comment by yourdoom9898** [comment]
@Poojawa Can't you just vore the expanded monkey? This just applies to chomping on the cube and having it expand in your chest

File: code/modules/food_and_drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -184,6 +184,15 @@
 	tastes = list("the jungle" = 1, "bananas" = 1)
 	foodtype = MEAT | SUGAR
 
+/obj/item/reagent_containers/food/snacks/monkeycube/On_Consume(mob/living/carbon/M)
+	if(iscarbon(M))
+		M.visible_message("[src] bursts out of [M]!</span>")
+		M.say("*scream")
```
Comment (Ghommie) [inline code/modules/food_and_drinks/food/snacks_meat.dm]:
```suggestion
		M.emote("scream")
```
What a blissful torture.

**Review by deathride58 — DISMISSED** [review]

**Review by deathride58 — APPROVED** [review]

**Pull request merged**